samples/outlook/65-delegates-and-shared-folders/get-shared-properties.yaml

```diff
@@ -61,6 +62,48 @@ script:
             }
           });
         }
+
+        function runOnAppointment() {
+          if (!Office.context.mailbox.item.getSharedPropertiesAsync) {
+            console.error("Try this sample on an appointment from a shared folder.");
+            return;
+          }
+
+          Office.context.mailbox.getCallbackTokenAsync({ isRest: true }, function(result) {
+            if (result.status === Office.AsyncResultStatus.Succeeded && result.value !== "") {
+              Office.context.mailbox.item.getSharedPropertiesAsync(
+                {
+                  // Pass auth token along.
+                  asyncContext: result.value
+                },
+                function(result2) {
+                  let sharedProperties = result2.value;
+                  let delegatePermissions = sharedProperties.delegatePermissions;
+
+                  // Determine if user can do the expected operation, e.g., do they have Write permission?
```

Comment:
I guess this is existing on the one you're copying from... but this comment is a bit confusing - we say "Write" but really check for Read permissions in the code below. Update the comment to "Read" too?

Reply:
Updated. Thx.